src/launchpad/models/__init__.py

```diff
@@ -47,4 +48,9 @@
     "AndroidAppInfo",
     "AndroidBinaryAnalysis",
     "AndroidAnalysisResults",
+    # Range mapping models
+    "BinaryTag",
+    "Range",
+    "RangeConflict",
+    "RangeMap",
 ]
```

Comment:
(just my inner monologue) my unfamiliarity with python is shining here, but I'm guessing at some point we'll want to separate some of these into separate groups